s2i/builder.go

```diff
@@ -87,7 +93,7 @@ func WithPlatform(platform string) Option {
 
 // NewBuilder creates a new instance of a Builder with static defaults.
 func NewBuilder(options ...Option) *Builder {
-	b := &Builder{}
+	b := &Builder{name: DefaultName}
```

Comment:
As with buildpack builder, the above `var DefaultName` could be removed, I believe.
```suggestion
	b := &Builder{name: builders.S2I}
```